quickstart/201-azfw-with-secure-hub/provider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.69.0"
```

Comment:
Why are we tying this to the specific version? Unless there is a specific reason, this should be for `version = "~>3.0"`

Reply:
I prefer do give users the latest versions this was tested on so they dont run into any issues but happy to make the change.

Reply:
> I prefer do give users the latest versions this was tested on so they dont run into any issues but happy to make the change.

@cshea15 Once integrated into the system, each sample is automatically tested with each release of Terraform and the Terraform Azure provider (generally weekly). This is how we catch breaking changes so that we can update the code and article without relying on customers to tell us of broken articles.

Reply:
got it updated to the suggested versioning.